scripts/mktcli.js

```diff
@@ -168,36 +210,38 @@ const main = {
       )
 
     program
-      .command('publication-cancel <coord>')
-      .description('Cancel a publication (x,y)')
+      .command('publication-cancel <assetId> <assetType> <contractName>')
+      .description(
+        'Cancel a publication. `contractName` defaults to `Marketplace`'
+      )
       .action(
-        asSafeAction(async coord => {
-          const [x, y] = parseCLICoords(coord)
-          const tokenId = await Parcel.encodeTokenId(x, y)
-          const contract = eth.getContract('Marketplace')
+        asSafeAction(
+          async (assetId, assetType, contractName = 'Marketplace') => {
+            const asset = await getAssetFromCLIArgs(assetId, assetType)
+            const contract = eth.getContract(contractName)
 
-          await unlockAccountWithPrompt()
-          const txHash = await contract.cancelOrder(tokenId)
+            await unlockAccountWithPrompt()
+            const txHash = await contract.cancelOrder(asset.token_id)
 
-          log.info(`(publication-cancel) coords:(${x},${y})`)
-          log.info(`(publication-cancel) tx:${txHash}`)
-        })
+            log.info(`(publication-cancel) id:(${asset.id})`)
+            log.info(`(publication-cancel) tx:${txHash}`)
+          }
+        )
       )
 
     program
-      .command('blockchain-events <coord>')
+      .command('blockchain-events <assetId> <assetType>')
       .option('--show-table-header', 'Show the format for each row')
       .option('--show-tx-hash', 'Show the transaction hash')
-      .description('Get chronological blockchain events of a (x,y) coordinate')
+      .description('Get chronological blockchain events of an asset')
       .action(
-        asSafeAction(async (coord, options) => {
+        asSafeAction(async (assetId, assetType, options) => {
+          const asset = await getAssetFromCLIArgs(assetId, assetType)
           const events = BlockchainEvent.getEvents()
-          const [x, y] = parseCLICoords(coord)
-          const tokenId = await Parcel.encodeTokenId(x, y)
 
           const blockchainEvents = await BlockchainEvent.findByArgs(
             'assetId',
-            tokenId
+            asset.token_id
```

Comment:
Should we use `asset.id` instead? Doubt because of:  https://github.com/decentraland/marketplace/pull/588/files#diff-14c145377d6748040b73ee576c8e1d57R202

Reply:
it's kind of confusing, but blockchain_events only deals with blockchain data it knows nothing about db `ids`